### ISSUE TITLE
GTEST/ucs/test_event_set.cc compile error: ignoring return value

### ISSUE THREAD

**Issue by tonycurtis** [openucx/ucx#3590]
Ubuntu 19.04, GCC 8.3.0

git master @ e634b23a67ecc29b368b223ef6c434532c45f532

```
make[3]: Entering directory '/home/tc/ucx-gtest/source/test/gtest'
g++ -DHAVE_CONFIG_H -I. -I../..  -DCPU_FLAGS="" -I/home/tc/ucx-gtest/source/src -I/home/tc/ucx-gtest/source -I/home/tc/ucx-gtest/source/src -I/opt/knem/include -I../../src -I../../test -I../../src -I../../test/gtest  -fopenmp   -I/usr/include -O3 -g -Wall -Werror  -fno-tree-vectorize -DGTEST_UCM_HOOK_LIB_DIR="\"/home/tc/ucx-gtest/source/test/gtest/ucm/test_dlopen/.libs\""  -MT ucs/gtest-test_event_set.o -MD -MP -MF ucs/.deps/gtest-test_event_set.Tpo -c -o ucs/gtest-test_event_set.o `test -f 'ucs/test_event_set.cc' || echo './'`ucs/test_event_set.cc
ucs/test_event_set.cc: In static member function ‘static void* test_event_set::event_set_read_func(void*)’:
ucs/test_event_set.cc:25:14: error: ignoring return value of ‘ssize_t write(int, const void*, size_t)’, declared with attribute warn_unused_result [-Werror=unused-result]
         write(fd[1], evfd_data, strlen(test_event_set::evfd_data));
         ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1plus: all warnings being treated as errors
make[3]: *** [Makefile:1813: ucs/gtest-test_event_set.o] Error 1
```

**Comment by tonycurtis** [comment]
PR #3592

**Comment by shamisp** [comment]
fixed in #3582
